src/components/overlays/FlyTooltip/FlyTooltip.scss

```diff
@@ -6,7 +6,7 @@
 	display: inline-block;
 
 	&:hover,
-	&.FlyTooltip_Container__ForceHoverState {
+	&.FlyTooltip_Container__forceShowState {
```

Comment:
This is such a nit, but can you capitalize the 'f' for the class?

Reply:
Done!